test/CMakeLists.txt

```diff
@@ -187,6 +187,25 @@ if(MIGRAPHX_ENABLE_PYTHON)
     add_subdirectory(py)
 endif()
 
+# multitarget test
+if(MIGRAPHX_ENABLE_GPU AND MIGRAPHX_ENABLE_CPU AND MIGRAPHX_ENABLE_FPGA)
```

Comment:
Since FPGA is still missing it would be better to have tests without needing this enabled for now.

We could also have unit tests that can run when no target is enabled. We can just create different targets for unit tests, but maybe that is too much work.

Reply:
> We can just create different targets for unit tests, 

Yeah. That would require creating some example passes for that target as well. 

>  when no target is enabled 

I have unit-test with single target and also have unit-test for the `run_on_target()` operation.